rust/crd/src/lib.rs

```diff
@@ -24,7 +24,7 @@ use stackable_operator::{
     k8s_openapi::{api::core::v1::Volume, apimachinery::pkg::api::resource::Quantity},
     kube::{runtime::reflector::ObjectRef, CustomResource, ResourceExt},
     memory::{BinaryMultiple, MemoryQuantity},
-    product_config_utils::{ConfigError, Configuration},
+    product_config_utils::{Configuration, Error as ConfigError},
```

Comment:
We now went without renaming the error in other operators.

Reply:
What do you mean here? The error is not renamed, right? do I need to change anything?

Reply:
We use the "full qualified name" in other operators, its just for consistency e.g. like https://github.com/stackabletech/zookeeper-operator/blob/4b42039afc67a87d64b12860ed0d146dfa3cfcaa/rust/crd/src/lib.rs#L330